.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: 
+    - python
+
+python:
+    - "3.6"
+
+os: 
+    - linux
+
+branches:
+    only:
+    - metadata-manager
+
+install:
+    - pip install pipenv
+    - pipenv install .
+    - pip install flake8
+
+script:
+    - python3 -m  unittest discover tests
+    - flake8 --ignore=F401
```

Comment:
Why to ignore this?

Reply:
Was solved by fixing directory structure. And moving `attachmeta.py` from `src` to project root.
https://github.com/fossdelhi/sync-music/commit/d070f849ab383ced32b0e176237cee8d33a3a8fa

https://github.com/fossdelhi/sync-music/commit
/67ca86c217a0e5ce48745119939b1916efd83a18